roles/uchiwa/templates/uchiwa.json.j2

```diff
@@ -0,0 +1,14 @@
+ {
+   "sensu": [
+     {
+       "name": "{{ uchiwa_sensu_api_server_name }}",
+       "host": "{{ sensu_api_server }}",
+       "port": {{ sensu_api_port }} 
+     }
```

Comment:
Wouldn't it be better to support multiple data centers, eg. have the value of key "sensu" in defaults/main.yml, with the default value of the hash above?